### PR TITLE
README needed some changing

### PR DESCRIPTION
**Pull request by StupidRepo**
I just made it look a little bit better.

### CODE REVIEW

**Review by qreeves — CHANGES_REQUESTED** [review]
Rest seems fine.

File: readme.txt [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
-* Download for free at https://redeclipse.net/
+* Completely free; download [here](https://redeclipse.net/)
 
 ## About
 
 Building on over 14 years of gameplay design and the lessons learned from Red Eclipse 1.x, Red Eclipse 2.x+ focuses on agile, easily accessible arena shooter gameplay over adherence to antiquated arena shooter tropes. For creators, the full power of the [Tesseract](http://tesseract.gg/) engine is at your fingertips in the included realtime multiplayer editor.
 
-The project is a free and open source game built using SDL and OpenGL which allows it to be ported to many platforms; you can [download a package](https://redeclipse.net/download) for Windows, GNU/Linux, BSD, and macOS; or grab a development copy from our [Git repository](https://redeclipse.net/devel) and live on the bleeding edge.
+The project is a free and open source game built using OpenAL and OpenGL which allows it to be ported to many platforms; you can [download a package](https://redeclipse.net/download) for Windows, GNU/Linux, BSD, and macOS; or build a development copy by cloning the [Git repository](https://redeclipse.net/devel) and live on the bleeding edge.
```
[the remark this answers is not in the document]
Comment (qreeves) [reply readme.txt:13]: But feel free to add OpenAL to the list still.

File: readme.txt [hunk 2]
```diff
@@ -4,20 +4,20 @@
 * Features parkour gameplay, such as impulse boosts, dashing, and other tricks
 * Favourite gamemodes with an array of mutators and variables
 * A built-in editor lets you create your own release-quality maps cooperatively online
-* Download for free at https://redeclipse.net/
+* Completely free; download [here](https://redeclipse.net/)
```
[the remark this answers is not in the document]
Comment (StupidRepo) [reply readme.txt:7]: I don't either, but I didn't really like the original. It didn't fit in. Besides, a little below that it mentions that the game is free. So I guess I could remove it?

**Comment by StupidRepo** [comment]
I don't know why I did that, lol.

**Review by StupidRepo — COMMENTED** [review]
Hoping this makes that stupid warning go away

File: readme.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (qreeves) [reply readme.txt]: No, the top of the file is basically the "TLDR" version of what is below, and typically how games promote themselves online. In general, this file is supposed to be copy/pasted when setting up a new landing page.

File: readme.txt [hunk 3]
```diff
@@ -4,20 +4,19 @@
 * Features parkour gameplay, such as impulse boosts, dashing, and other tricks
 * Favourite gamemodes with an array of mutators and variables
 * A built-in editor lets you create your own release-quality maps cooperatively online
-* Download for free at https://redeclipse.net/
```
Comment (qreeves) [inline readme.txt:7]: Please reinstate this line.

**Review by qreeves — CHANGES_REQUESTED** [review]

**Pull request closed without merging**